.github/workflows/unity.yml

```diff
@@ -13,6 +13,7 @@ on:
       - 'lib/mac/**'
       - 'lib/windows/**'
       - 'resources/**'
+      - '!resources/lint/**'
```

Comment:
I know it's annoying, but since we went to the trouble above, we might as well do `audio_samples`, `contexts*/<supported_platform>` and `test`. Currently the unity test would run if we changed the beaglebone resources, which no one wants haha...